int64 a
---
bool result